### ISSUE TITLE
options.collectTypes doesn't support "cy.intercept"

### ISSUE THREAD

**Issue by thaiching** [archfz/cypress-terminal-report#113]
Hi,
   First of all thank you for creating this awesome plugin. I've been searching for a plugin to output all the Cypress logs to terminal and this plugin is doing exactly what I needed. :)

   In the index.ts plugin file, I've specified the type of logs that I want to output to the terminal using the following lines:

```
const options = {
  collectTypes: ["cy:log", "cy:xhr", "cy:request", "cy:route", "cy.intercept", "cy:command"],
};

require('cypress-terminal-report/src/installLogsCollector')(options);
```
However when I run my tests, the following error is thrown. It doesn't look like the **collectTypes** is supporting "cy.intercept"? However the documentation does mention the support of cy.intercept for collectTypes.

When I removed "cy.intercept", it works fine. 

<img width="880" alt="Untitled" src="https://user-images.githubusercontent.com/7057586/128376626-f40fd6e8-012a-415d-951f-07bce4b447de.png">

**Comment by archfz** [comment]
Its cy:intercept not cy.intercept

**Comment by thaiching** [comment]
Hi @archfz , 
   Sorry for the typo. I've also tried "cy:intercept" but I'm getting an error as well.

<img width="956" alt="Untitled" src="https://user-images.githubusercontent.com/7057586/128513246-5b58e358-5096-42d4-acff-55e75470e845.png">

**Comment by archfz** [comment]
Released in 3.3.1

**Comment by thaiching** [comment]
Thank you!
